Cut release 0.70 (#6134)
prometheus-operator/prometheus-operator: CHANGELOG.md

## 0.70.1 / 2023-11-30

* [CHANGE] Use camelCase on scrapeConfig in prometheus job names. #6124
* [CHANGE] Update json tags for ConsulSD configs. #6094
* [FEATURE] Support service,pod,endpoints,endpointslice,ingress roles to KubernetesSDConfigs in ScrapeConfig CRD. #6089

> * [FEATURE] Add `trackTimestampsStaleness` limit to CRDs. #6105

Reviewer: ```suggestion
* [FEATURE] Add `trackTimestampsStaleness` limit to ServiceMonitor, PodMonitor and ScrapeConfig CRDs. #6105
```

* [ENHANCEMENT] Expose flag to enable the reloader probes in the jsonnet configuration. #6075

> * [ENHANCEMENT] Send logs to `os.Stdout`. #6085

Reviewer: ```suggestion
* [ENHANCEMENT] Config reloader now sends all logs to `os.Stdout`. #6085
```

* [BUGFIX] Remove verbose logging in admission-webhook pods. #6095

> * [BUGFIX] Skip adding shard relabel rules automatically if already set. #6099

Reviewer: ```suggestion
* [BUGFIX] Skip adding shard relabel rules if they were already set in `additionalScrapeConfigs`. #6099
```

> * [BUGFIX] Check for nil in MS Teams config. #6109

Reviewer: ```suggestion
* [BUGFIX] Fix nil pointer exception when title and text are not set in MS Teams config. #6109
```

> ## 0.70.1 / 2023-11-30

Reviewer 2: ```suggestion
## 0.70.0 / 2023-11-30
```
